src/definitions/elements/input.less

```diff
@@ -480,7 +480,8 @@
             padding-right: @labeledMargin;
         }
     }
-    .ui.ui[class*="corner labeled"]:not([class*="left corner labeled"]).icon:not([class*="left icon"]).input > .ui.dropdown {
+    .ui.ui[class*="corner labeled"]:not([class*="left corner labeled"]).icon:not([class*="left icon"]).input
+        > .ui.dropdown {
```

Comment:
The indentation here is confusing. The > not starts at the same position as the next & . To me this is more confusing than the previous formatting